### PR TITLE
Locked dashboard

### PR DESCRIPTION
**Pull request by NadaaFarook**

### CODE REVIEW

File: client/src/pages/dashboard/index.tsx [hunk 1]
```diff
@@ -67,11 +67,11 @@ function Dashboard() {
   useEffect(() => {
     switch (status?.status) {
```
Comment (uditkumar01) [inline client/src/pages/dashboard/index.tsx:68]: comment the whole switch case if not needed

**Review by uditkumar01 — COMMENTED** [review]
Just have one suggestion, please have a look. Rest everything is perfect 🔥

File: client/src/pages/dashboard/index.tsx (hunk 1 above)
Comment (NadaaFarook) [reply client/src/pages/dashboard/index.tsx:68]: Its needed. For now , the dashboard is locked , we might allow portfolio submission once again

**Comment by uditkumar01** [comment]
@OmkarK45 @NadaaFarook can't make PR against main branch as her branch is currently updated with all the changes on develop.

**Comment by OmkarK45** [comment]
Merging this as it has been reviewed 👍

**Pull request merged**